uc-crux-llvm/src/UCCrux/LLVM/Main.hs

```diff
@@ -219,7 +219,7 @@ translateLLVMModule ::
   IO SomeModuleContext'
 translateLLVMModule llOpts halloc memVar moduleFilePath llvmMod =
   do
-    Some trans <-
+    (Some trans, _warns) <- -- TODO? should we do something with these warnings?
```

Comment:
What do you think about this, @langston-barrett? Should `uc-crux-llvm` emit warnings discovered during translation?